### ISSUE TITLE
Low contrast in yellow alert component

### ISSUE THREAD

**Issue by Niksac** [zuramai/mazer#76]
While using this theme, I noticed that the contrast in the yellow alert box (and other components that share this color scheme) is pretty low.

What's your opinion on this? Maybe we could use some color scheme for the new release.

![image](https://user-images.githubusercontent.com/1611565/166101424-4bc81fbc-4d5a-4748-8b07-e647d2aea9e6.png)

**Comment by zuramai** [comment]
Yes, at the first I was concerning about the correct color of yellow.
It will be good to find the better one for next release.

**Comment by zuramai** [comment]
![image](https://user-images.githubusercontent.com/45036724/167649388-884e6133-c310-422f-8e8f-0035a5b8343e.png)

The colour contrast is now WCAG AAA [7e89a1d](https://github.com/zuramai/mazer/commit/7e89a1db7f0b683862303c509bd4b00c5b38d0c2) ✅
